week1/src/components/headercomponent.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const ComponentHeader=(props)=>{
+        return (
+        <div>
+          <h2 className="Componentheader">{props.title} </h2>
+        </div>
+        )
+        }
```

Comment:
some ugly formatting here, are you using prettier?

Reply:
my laptop computer broke down. I think there is no prettier new one.I will take care